guides/common/modules/proc_importing-a-subscription-manifest-into-foreman-server.adoc

```diff
@@ -16,7 +16,7 @@ Importing a manifest does not change your organization's Simple Content Access s
 .Prerequisites
 * You must have a Red{nbsp}Hat Subscription Manifest file exported from the https://access.redhat.com[Red{nbsp}Hat Customer Portal].
 ifndef::orcharhino[]
-For more information, see https://access.redhat.com/documentation/en-us/red_hat_subscription_management/2022/html/using_red_hat_subscription_management/using_manifests_con[Using Manifests] in _Using Red Hat Subscription Management_.
+For more information, see https://access.redhat.com/documentation/en-us/subscription_central/2023/html/creating_and_managing_manifests_for_a_connected_satellite_server/index[Using Manifests] in _Using Red Hat Subscription Management_.
```

Comment:
Since the BZ wants the link to go to Using Manifests, I suggest changing the section reference to _Using manifests for a disconnected Satellite Server_